assembly/index.ts

```diff
@@ -0,0 +1,203 @@
+
+import { base64, runtime_api } from "near-runtime-ts";
+import { JSONEncoder as _JSONEncoder, JSON } from "assemblyscript-json";
+import { u128 } from "bignum";
+
+// Runtime functions
+// tslint:disable: no-unsafe-any
+
+@global
+class JSONEncoder extends _JSONEncoder {
+  encode<T>(name: string, val: T): void {
+    encode<T, JSONEncoder>(val, name, this);
+  }
+}
+
+type Usize = u64;
+//@ts-ignore
+@global
+function read_register(register_id: Usize, ptr: Usize): void {
```

Comment:
why are these wrapper functions are needed? is there any way to import them directly, e.g. like:

```js
import { register_len } from "near-runtime-ts/runtime_api";
```

Reply:
They will be inlined.  The idea was to not have to add any imports to the sources, but rather have global definitions since this file is compiled alongside the main file.